docs/source/explicit_comms.rst

```diff
@@ -11,7 +11,6 @@ implementation of the Dataframe `shuffle <https://github.com/rapidsai/dask-cuda/
 Usage
 -----
 
-In order to use explicit-comms in Dask/Distributed automatically, simply define the environment variable ``DASK_EXPLICIT_COMMS=True`` or setting the ``"explicit-comms"``
-key in the `Dask configuration <https://docs.dask.org/en/latest/configuration.html>`_.
+In order to use explicit-comms in Dask/Distributed for shuffle-based DataFrame operations, simply pass the ``shuffle="explicit-comms"`` key-word argument to the appropriate DataFrame API (e.g. ``ddf.sort_values(..., shuffle="explicit-comms")```). You can also change the default shuffle algorithm to explicit-comms by define the ``DASK_SHUFFLE=explicit-comms`` environment variable, or by setting the ``"shuffle"`` key to ``"explicit-comms"`` in the `Dask configuration <https://docs.dask.org/en/latest/configuration.html>`_.
```

Comment:
@madsbk - Does this change seem okay to you? My understanding is that the `EXPLICIT_COMMS` config/env-variable was only being used for the shuffle/rarrange_by_column_tasks patch.  As far as I can tell, we never use this config to automatically use the explicit-comms merge anywhere. Is that correct?

Reply:
Yes, this is correct.